pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[project]
+name = "spiceypy"
+version="4.0.2"
+author="Andrew Annex"
+author_email="ama6fy@virginia.edu"
+description="A Python Wrapper for the NAIF CSPICE Toolkit"
+license = "MIT"
+homepage = "https://github.com/AndrewAnnex/SpiceyPy"
+repository = "https://github.com/AndrewAnnex/SpiceyPy"
+documentation = "https://spiceypy.readthedocs.io"
+keywords=["spiceypy", "spice", "naif", "jpl", "space", "geometry", "ephemeris"]
+readme = "README.rst"
+classifiers=[
+    "Development Status :: 5 - Production/Stable",
+    "Natural Language :: English",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Astronomy",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: POSIX :: BSD :: FreeBSD",
+    "Operating System :: Microsoft :: Windows",
+]
+requires-python = ">=3.6, <4"
+dependencies = ["numpy>=1.17.0"]
+
+[build-system]
+requires = ["setuptools >= 42", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.cibuildwheel]
+# Only build on CPython 3.9
+build = "cp39-*"
+# Skip 32-bit builds and pypy
+skip = ["*-win32", "*-manylinux_i686", "pypy*"]
+# other options
+build-verbosity = 2
+before-build = "python get_spice.py"
+before-test = "pip install -r ci-requirements.txt"
+test-command = "pytest --pyargs {project}"
+
```

Comment:
add an after test step `CIBW_AFTER_TEST=cd {output_dir} && coverage combine --append {package_dir}` to enable coverage reports https://github.com/pypa/cibuildwheel/issues/115